proposals/0117-configurable-time-before-shutdown.md

```diff
@@ -51,16 +28,15 @@ So this problem can be solved only with modifications on SDL side.
 
 ```
 // Write all logs in queue to file system before shutdown 
-FlushLogMessagesBeforeShutdown = false
+FlushLogMessagesBeforeShutdown = true
 
 // Maximum time to wait for writing all data before exit SDL in seconds
 MaxTimeBeforeShutdown = 30
 ```
 
-By default `FlushLogMessagesBeforeShutdown` should be false. In that case, SDL should not wait for all data to be written to the file system and stop process after receiving `OnExitAllApplications` notification. 
-
-`MaxTimeBeforeShutdown` would be used in case if `FlushLogMessagesBeforeShutdown` is `true`. It should measure time from `OnExitAllApplications` notification received. In case if writing logs to file system takes more time than specified, SDL should terminate writing and finish process. 
+By default `FlushLogMessagesBeforeShutdown` should be `true` to keep existing SDL behavior unchanged, so SDL will be able to dump all the log messages into the file system as it currently doing. This flag can be changed to `false` in the case when missing log messages before the shutdown are not necessary (for example in the production builds). In this case, SDL can be closed much faster.
```

Comment:
I guess coma can be omitted in `In this case, SDL can ...`

Reply:
@dboltovskyi fixed in https://github.com/LuxoftSDL/sdl_evolution/pull/10/commits/020306a3a8e5f514ac2d89f0697fd8c53b3d3c8b